### ISSUE TITLE
cli_code() output to terminal

### ISSUE THREAD

**Issue by krlmlr** [r-lib/cli#127]
gains newlines between the code lines when copy-pasting (and also an extra space in front of each line of code).

Steps to reproduce:

1. Run `cli::cli_code(format(cli::cli_blockquote))` in RStudio
2. Copy the output to the clipboard
3. Paste to the console

I'm using it to print code to the console that is supposed to be copy-pasted. Do we need a "copy-paste-friendly" mode for `cli_code()`?

**Comment by gaborcsardi** [comment]
I cannot reproduce this.

> and also an extra space in front of each line of code)

The default theme for code has a left margin.

**Comment by krlmlr** [comment]
It's not the newlines -- the text has spaces which make the text too wide when copy-pasting:

```r
nchar(capture.output(cli::cli_code(letters[1:3])))
## [1] 76 76 76 76 76
```

(Not reprexable.)

Maybe we can reduce the padding, or make the padding as wide as necessary?

**Comment by gaborcsardi** [comment]
Yeah, we can remove the padding. RStudio does not change the bg color any more, anyway.

**Comment by gaborcsardi** [comment]
Should be fixed in d6daa153753a77b2dbfb640668f8c3301e0a204d
